pkg/daemon/ceph/client/filesystem.go

```diff
@@ -173,6 +174,11 @@ func IsMultiFSEnabled() bool {
 	}
 	return false
 }
+func SetPools(context *clusterd.Context, cephVersion cephver.CephVersion, clusterName, fsName string, metdataPool []*model.Pool, dataPool *model.Pool) error {
+	args := []string{"osd", "pool", "set", fsName, "size"}
```

Comment:
So this is the command that is needed to update the pool size, the only problem that I'm getting is that I'm not able to use `len(metadataPool)`, I know it's due to the data type mismatch.
@travisn can you please add some suggestions?.

Reply:
hope this helps https://gobyexample.com/arrays @rajatsingh25aug